src/js/data/ui-update/utils.ts

```diff
@@ -0,0 +1,18 @@
+const LOCAL_DEV_VERSION = "0.0.0-dev+semantic-release";
```

Comment:
do you think it would make sense to merge this with `plugins/services/src/js/utils/Version.ts` some time?